src/js/index.js

```diff
@@ -0,0 +1,75 @@
+const espressoMenuForm = document.querySelector("#espresso-menu-form");
+const menuList = document.querySelector("#espresso-menu-list");
+
+function App() {
+  const addMenu = function(espressoNewMenu) {
+      const menuTemplate = `<li class="menu-list-item d-flex items-center py-2">
+      <span class="w-100 pl-2 menu-name">${espressoNewMenu}</span>
+      <button
+        type="button"
+        class="bg-gray-50 text-gray-500 text-sm mr-1 menu-edit-button"
+      >
+        수정
+      </button>
+      <button
+        type="button"
+        class="bg-gray-50 text-gray-500 text-sm menu-remove-button"
+      >
+        삭제
+      </button>
+    </li>`
+
+      menuList.insertAdjacentHTML("beforeend", menuTemplate);
+  }
+
+  const updateMenuCount = function() {
+    const updateCount = document.querySelector(".menu-count");
+    updateCount.innerText = `총 ${menuList.children.length} 개`;
+  }
+
+  const registerMenu = function(e) {
+    e.preventDefault();
+    const espressoMenuAdd = espressoMenuForm.querySelector("#espresso-menu-name");
+    const espressoNewMenu = espressoMenuAdd.value;
+    if (espressoMenuAdd.value.trim() === ""){
+        alert("메뉴를 입력해주세요!");
+        return;
+    }
+    espressoMenuAdd.value = "";
+    addMenu(espressoNewMenu);
+    updateMenuCount();
+}
+
+const updateMenuName = function(e) {
+    const espressoMenuRename = e.target.closest("li").querySelector("span.menu-name");
+    const renamedMenu = prompt("메뉴를 수정 하세요", espressoMenuRename.innerText);
+    if (renamedMenu != null && renamedMenu.trim().length != 0){
+      espressoMenuRename.innerText = renamedMenu;
+    }
```

Comment:
```suggestion
    if (renamedMenu != null && renamedMenu.trim()){
      espressoMenuRename.innerText = renamedMenu;
    }
```

조건문이 과한 것처럼 보여요. trim()까지만 사용해도 조건을 충족시킬 수 있지 않을까요?

Reply:
```suggestion
    if (!renamedMenu.trim()){
      espressoMenuRename.innerText = renamedMenu;
    }
```

위와 같이도 처리 가능하지 않나요?